src/routes/feeds/index.ts

```diff
@@ -0,0 +1,46 @@
+import express, { Request, Response } from 'express';
+import { createClient } from 'redis';
+import axios from 'axios';
+import transactionWrapper from '../transactionWrapper';
+import authMiddleware from '../../middleware/authMiddleware';
+import { generateRedisConfig } from '../../lib/queue';
+
+const {
+  uri: redisUrl,
+  tlsEnabled,
+} = generateRedisConfig();
+
+const feedUrl = 'https://acf-ohs.atlassian.net/wiki/createrssfeed.action?types=page&pageSubTypes=comment&spaces=conf_all&title=Confluence+RSS+Feed&labelString=releasenotes&excludedSpaceKeys%3D&sort=modified&maxResults=10&timeSpan=5&showContent=true&confirm=Create+RSS+Feed&os_authType=basic';
```

Comment:
I think it's totally fine to hardcode this in there but we will obviously need to expand this API to accommodate different content labels in Confluence